components/genesys-email-flow/dev/main.tf

```diff
@@ -0,0 +1,44 @@
+terraform {
+  backend "local"  {
+      path ="/Users/johncarnell/genesys_terraform/carnell1_dev/tfstate"
```

Comment:
Either we add an instruction to change the backend local path or just remove it entirely so the state will be saved in the same directory.

Reply:
See lines 71-83 of the genesys-email-flow/README.md.  I have the comments in there for setting this directory.